test/integration/client-side-operations-timeout/client_side_operations_timeout.prose.test.ts

```diff
@@ -216,9 +289,35 @@ describe('CSOT spec prose tests', function () {
        *   - Expect this to fail with a timeout error.
        * 1. Verify that a `find` command was executed against the `keyvault.datakeys` collection as part of the `encrypt` call.
        */
+
+      afterEach(async function () {
+        await clearFailPoint(this.configuration);
+      });
+
+      it('throws a timeout error', metadata, async function () {
```

Comment:
Did you mean to implement this prose test?  You didn't make the encrypt() helper respect timeoutMS.  I'm surprised this is passing 🤔

Reply:
Same with decrypt.

Reply:
LOL oops... 🤔